src/rust/engine/.cargo/config

```diff
@@ -0,0 +1,7 @@
+# tokio_unstable is necessary for access to the `id`-related functions of tokio::task::JoinSet.
+# See https://docs.rs/tokio/1.21.1/tokio/task/struct.JoinSet.html#method.join_next_with_id.
+#
+# tokio_no_const_thread_local is necessary to avoid a clippy lint related to the use of task_local!
+# at stdio/src/lib.rs:420:1.
```

Comment:
Could the clippy lint be skipped instead? Seems like it might be easier to remove when it becomes stale.

Reply:
I tried to attaching the `#[allow()]` to the `task_local!` and the definition within the macro's block to no avail. The lint is being caused by code generated by the macro.

Reply:
From discussion on Tokio Discord, the lint should go away on Rust 1.64.0. So doing that upgrade first.